tests/init/test_init.py

```diff
@@ -685,6 +707,57 @@ def test_init_with_custom_env(tmp_path_factory: TempPathFactory) -> None:
     )
 
 
+def test_init_fullstack_template_fails_on_missing_python(
```

Comment:
This isn't technically the fullstack template.

You could also add a comment to indicate that the fullstack template is an example of a template which executes python as part of running copier.

```suggestion
def test_init_template_with_python_fails_on_missing_python(
```